New module named hosts for managing entries in /etc/hosts

resmo opened the pull request:
Quite simple but useful module for managing entries in /etc/hosts.

Tested on Ubuntu 12.04.

resmo committed:
added module hosts for managing entries in /etc/hosts

Signed-off-by: René Moser <mail@renemoser.net>

mpdehaan commented:
The code and idea here is fine, however including this sets a precedent as pretty soon we'd have a module for every sort of configuration file, and in many cases, users can just template the same known host file to all systems, in other cases, they could use line-in-file to add new entries.

I am thinking we should not include this one for that reason.

resmo commented:
I understand your concerns, and your stance makes user love ansible. Keep it simple.

resmo commented:
@mpdehaan 
I updated this module to even be more useful, I added aliases and you can now define a hostname to be absent, not only an IP)

https://github.com/resmo/ansible-host/blob/master/host

I just wanted to let you know and wanted you to look at it again and if it still can not be in ansible core. 

The possibility to managed host entries on its own is quite common, other (not that easy) config management solutions also have a resource for it (http://docs.puppetlabs.com/references/latest/type.html#host). 

People can still decide to use this module or manage host entries using templates of course, but I think it is simple, easy and common enough to be in ansible core.

Thanks in advance.
